src/zos_jobs/zowe/zos_jobs_for_zowe_sdk/jobs.py

```diff
@@ -117,6 +117,22 @@ def delete_job(self, jobname, jobid, modify_version="2.0"):
         response_json = self.request_handler.perform_request("DELETE", custom_args, expected_code=[202, 200])
         return response_json
 
+    def _issue_job_request(self, req: dict, jobname: str, jobid: str, modify_version):
+
+        custom_args = self._create_custom_request_arguments()
+        job_url = "{}/{}".format(jobname, jobid)
+        request_url = "{}{}".format(self.request_endpoint, job_url)
+        custom_args["url"] = request_url
+        custom_args["json"] = {
+            **req,
+            "version": modify_version 
+        }
+
+        custom_args["headers"]["X-IBM-Job-Modify-Version"] = modify_version
+            
+        response_json = self.request_handler.perform_request("PUT", custom_args, expected_code=[202, 200])
+        return response_json
+       
     def change_jobs_class(self, jobname: str, jobid: str, class_name: str, modify_version="2.0"):
```

Comment:
Could you please rename the method to `change_job_class` ("job" singular)? Sorry for being picky 😢

Reply:
Done with changes :relaxed: